### PR TITLE
Business info page

### PR DESCRIPTION
**Pull request by anand-ts**
- Added business info page

### CODE REVIEW

**Review by clarkcousins — APPROVED** [review]
lgtm

File: .gitignore
Comment (DavidHe137) [inline .gitignore:1]: Should not ignore package.json, as this file is necessary for other collaborators to know which npm modules are needed.

**Review by DavidHe137 — CHANGES_REQUESTED** [review]
Adjust .gitignore please

File: .gitignore
Comment (anand-ts) [reply .gitignore:1]: Changed!

**Review by sohumraoGT — APPROVED** [review]
LGTM

**Review by achen443 — APPROVED** [review]

**Review by willshutt — APPROVED** [review]
lgtm, great work on this

**Review by anand-ts — COMMENTED** [review]
LGTM!

**Review by DavidHe137 — COMMENTED** [review]
good

**Pull request merged**